stix_shifter_modules/arcsight/stix_translation/json/to_stix_map.json

```diff
@@ -0,0 +1,1515 @@
+        "object": "process",
+        "references": "parent_process"
+      }
+    ],
+    "destinationUserId": [
+      {
+        "key": "user-account.user_id",
+        "object": "dst_user"
+      },
+      {
+        "key": "process.creator_user_ref",
+        "object": "process",
+        "references": "dst_user"
+      }
+    ],
+    "sourceUserId": [
+      {
+        "key": "user-account.user_id",
+        "object": "src_user"
+      },
+      {
+        "key": "process.creator_user_ref",
+        "object": "parent_process",
+        "references": "src_user"
+      }
+    ],
+    "sourceUserName": {
+      "key": "x-arcsight-event-source.source_username",
+      "object": "source"
+    },
+    "destinationUserName": {
+      "key": "x-arcsight-event-destination.destination_username",
+      "object": "destination"
+    },
+    "fileName": {
+      "key": "file.name",
+      "object": "file"
+    },
+    "filePath": [
+      {
+        "key": "directory.path",
+        "object": "file_dir",
+        "transformer": "ToDirectoryPath"
+      },
+      {
+        "key": "file.parent_directory_ref",
+        "object": "file",
+        "references": "file_dir"
+      }
+    ],
+    "MD5": {
+      "key": "file.hashes.MD5",
+      "object": "file"
+    },
+    "SHA1": {
+      "key": "file.hashes.SHA-1",
+      "object": "file"
+    },
+    "SHA256": {
+      "key": "file.hashes.SHA-256",
+      "object": "file"
+    },
+    "registry_data": {
+      "key": "windows-registry-key.values",
+      "object": "reg_key",
+      "transformer": "ArcsightToRegistryValue"
+    },
+    "registry_key" : {
+        "key": "windows-registry-key.key",
+        "object": "reg_key",
+        "transformer": "ArcsightToRegistryKey"
+    },
+    "categorySignificance": [
+      {
+        "key": "x-arcsight-event-category.category_significance",
+        "object": "event_category"
+      },
+      {
+	    "key": "x-ibm-finding.finding_type",
+        "object": "logger_finding"
+	  },
+      {
+        "key": "x-ibm-finding.event_category_details_ref",
+        "object": "logger_finding",
+        "references": "event_category"
+      }
+    ],
+    "categoryBehavior": {
+      "key": "x-arcsight-event-category.category_behavior",
+      "object": "event_category"
+    },
+    "categoryDeviceGroup": {
+      "key": "x-arcsight-event-category.category_device_group",
+      "object": "event_category"
+    },
+    "categoryObject": {
+      "key": "x-arcsight-event-category.category_object",
+      "object": "event_category"
+    },
+    "categoryOutcome": {
+      "key": "x-arcsight-event-category.category_outcome",
+      "object": "event_category"
+    },
+    "categoryTechnique": {
+      "key": "x-arcsight-event-category.category_technique",
+      "object": "event_category"
+    },
+    "eventId": [
+      {
+      "key": "x-arcsight-event.event_id",
+      "object": "event"
+      },
+      {
+        "key": "x-ibm-finding.event_details_ref",
```

Comment:
The to-stix mapping has a lot of extra references on the finding object. `x-ibm-finding` object should only have the following references:
`src_ip_ref` and `dst_ip_ref` (Reference to source and destination ipv4 or ipv6 cybox object)
`src_os_ref` and `dst_os_ref` (Reference to software cybox object representing the source or destination operating system)
`src_application_ref` and `dst_application_ref` (Reference to software cybox object representing the source or destination application)
`src_application_user_ref` and `dst_application_user_ref` (Reference to user-account cybox object representing the source or destination application user)
`src_database_user_ref` and `dst_database_user_ref` (Reference to user-account cybox object representing the source or destination database user)
`src_os_user_ref` and `dst_os_user_ref` (Reference to user-account cybox object representing the source or destination OS user)

Reply:
@delliott90  The extra references on the findings object are now removed. The changes are updated to PR branch.

Additionally, apart from the mandatory fields, we have also mapped the following attributes in the findings object by referring the  the schema document for custom STIX cybox. (severity, src device, dst device, src location, dst location, src_ip_ref, dst_ip_ref). 

Kindly let us know for any issues, Thank you.